salt/matchers/compound_match.py

```diff
@@ -25,7 +37,6 @@ def match(tgt, opts=None, minion_id=None):
     if not opts:
         opts = __opts__
     nodegroups = opts.get("nodegroups", {})
-    matchers = salt.loader.matchers(opts)
```

Comment:
Did passing `opts` do anything that is now not accounted for?

Reply:
We're already passing `opts` directly into the function, so I think we're covered.